tests.py

```diff
@@ -19,7 +19,18 @@ def test_word_is_solved():
     
 def test_display_letters():
     assert display_letters("hello", "h") == "h _ _ _ _", "Should reveal only first letter"
-
+    assert display_letters("hello", "hello") == "h e l l o", "Should reveal entire word"
+    assert display_letters("hello", "wptrk") == "_ _ _ _ _", "Should reveal nothing of the word"
+    assert display_letters("hello john", "hlljoenh") == "h e l l o  j o h n", "Should reveal whole phrase"
```

Comment:
There should be 3 spaces, not 2. If every character is padded with a space, then a space padded with two spaces is 3 spaces.